lifemonitor/utils.py

```diff
@@ -57,7 +58,22 @@ def load_ro_crate_metadata(roc_path):
     file_path = os.path.join(roc_path, RO_CRATE_METADATA_FILENAME)
     with open(file_path) as data_file:
         logger.info("Loading RO Crate Metadata @ %s", file_path)
-        return json.load(data_file)
+        data = json.load(data_file)
+        logger.debug("RO Crate Metadata: %r", data)
+        return data
+
+
+def load_test_definition_filename(filename):
+    with open(filename) as f:
+        return json.load(f)
+
+
+def search_for_test_definition(roc_path, ro_crate_metadata: dict):
+    # first search on the root roc_path for a test_definition file
+    filename = os.path.join(roc_path, RO_CRATE_TEST_DEFINITION_FILENAME)
```

Comment:
As per the [current specs](https://github.com/crs4/life_monitor/wiki/Test-Metadata-Draft-Spec), the test metadata file (`test-metadata.json`) is expected under a `test` directory, not at the top level.

Reply:
Fixed by https://github.com/crs4/life_monitor/pull/32/commits/351ff8e8cc14778d738a07fe3259770ef53e755c